packages/connect/src/connect.ts

```diff
@@ -156,15 +163,9 @@ async function connect(
   const ethersProvider = getEthersProvider(ethereumProvider, network)
   const orgConnector = getConnector(connector, network)
 
-  // Two independent tasks in parallel are done here:
-  //  - Resolving the organization address.
-  //  - Calling .connect() on the connector.
-  const [orgAddress] = await Promise.all([
-    resolveAddress(ethersProvider, location),
-    orgConnector.connect?.(),
-  ])
+  const orgAddress = await resolveAddress(ethersProvider, location)
```

Comment:
One thing we are still missing is during the `getEthersProvider` we should manually create a WebSocket provider for Rinkeby because of the issue we saw that the default provider is not overriding our custom `ensAddress` correctly.

Reply:
Note that the example with rinkeby is not working for this reason.

Reply:
Open this issue on ethers repo: https://github.com/ethers-io/ethers.js/issues/959

Reply:
It’s working now, thanks for reporting the issue :pray:  https://github.com/aragon/connect/pull/135/commits/78c0c21acb831cc24556d0b73278a17c78b34768